### DISCUSSION TITLE
isHighlighted option for images does not work anymore in sanity studio v3

### DISCUSSION THREAD

**Discussion by intensr** [sanity-io/sanity#3664] [Sanity Studio Feedback]
```
{
  title: 'Poster',
  name: 'poster',
  type: 'image',
  options: {
    hotspot: true // <-- Defaults to false
  },
  fields: [
    {
      name: 'caption',
      type: 'string',
      title: 'Caption',
      options: {
        isHighlighted: false // <-- doesnt work anymore
      }
    },
  ]
}
```

The is highlighted functionality for the [image-type](https://www.sanity.io/docs/image-type) doesnt seem to work anymore in studio v3. it is not possible to hide a field anymore, each field is just shown in the same view as a regular object is displayed.

**Comment by kmelve** [comment]
Thanks for reporting! [I've filed the `isHighlighted ` bug to the team](https://github.com/sanity-io/sanity/issues/3665). 

By the way, the default value for `hotspot` has always been `false`.

**Reply by intensr** [reply]
thanks knut, oh yeah sorry i was lazy and just edited the example code without removing the irrelevant part
